testnet/shared-config-test.json

```diff
@@ -140,6 +154,20 @@
               "interior": "here"
             }
           }
+        },
+        {
+          "resourceId": "0x0000000000000000000000000000000000000000000000000000000000001100",
+          "type": "fungible",
+          "native": true,
+          "symbol": "sygUSD",
+          "decimals": 18,
+          "assetName": "SygmaUSD",
+          "xcmMultiAssetId": {
```

Comment:
@freddyli7 actually I believe this should be different?

Reply:
@MakMuftic sorry I totally missed this message. you are correct, for sygUSD on testnet, its xcmMultiAssetId should be:
```
{
    "concrete": {
        "parents": 1,
        "interior": "X2": {
             "Parachain": 1000,
             "GeneralKey": {
                 "length": 6,
                 "data": "0x7379675553440000000000000000000000000000000000000000000000000000"
             }
        }
    }
}
```